src/collective/cover/configure.zcml

```diff
@@ -10,8 +10,14 @@
   <i18n:registerTranslations directory="locales" />
   <five:registerPackage package="." />
 
-  <includeDependencies package="." />
+  <include package="collective.js.bootstrap" />
+  <include package="collective.js.galleria" />
+  <include package="plone.app.blocks" />
+  <include package="plone.app.dexterity" />
+  <include package="plone.app.iterate" />
+  <include package="plone.app.lockingbehavior" />
   <include package="plone.app.relationfield" />
+  <include package="plone.app.tiles" />
```

Comment:
Isn't it needed to include every dependency from [metadata.xml](https://github.com/collective/collective.cover/blob/master/src/collective/cover/profiles/default/metadata.xml#L5-L11)?

How about to both lists be equal?

Reply:
including a package in `configure.zcml` does one thing and including it in `metadata.xml` does another; a package in `metadata.xml` should be included in `configure.zcml` but the opposite is not true.